pkgs/applications/misc/keepassx/community.nix

```diff
@@ -1,23 +1,33 @@
 { stdenv, fetchFromGitHub, cmake, makeWrapper, qttools
-, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, libgpgerror, glibcLocales, libyubikey, yubikey-personalization, libXi, qtx11extras
+, curl
+, glibcLocales
+, libXi
+, libXtst
+, libargon2
+, libgcrypt
+, libgpgerror
+, libmicrohttpd
+, libyubikey
+, qtbase
+, qtx11extras
+, yubikey-personalization
+, zlib
 , withKeePassHTTP ? true
```

Comment:
This is missing options for `withKeePassBrowser` (requires a new dependency on `libsodium`), `withNetworking`, and `withSSHAgent` and their equivalent CMAKE flags.

To preserve the spirit of the existing configuration, `withKeePassBrowser` should likely default to `true`.